src/postgres/postgresConfig.ts

```diff
@@ -0,0 +1,49 @@
+import pg from 'pg';
+import { Env } from '../utils/env.js';
+import fs from 'fs';
+
+export default async function postgresConfig() {
+    const { Client } = pg;
+    const postgresHost = Env.optional("POSTGRES_HOST", "localhost");
+    const postgresPort = Env.optional("POSTGRES_PORT", "5432");
+    const postgresUser = Env.optional("POSTGRES_USER", null);
+    const postgresPassword = Env.optional("POSTGRES_PASSWORD", null);
+    const postgresDatabase = Env.optional("POSTGRES_DB", null);
```

Comment:
should we put more useable defaults here? If one were to run postgres locally from scratch, what would the defaults be? Database `postgres` is probably all that's needed?

Reply:
It would really depend on how you run your instance. Eg. if you are using Docker then you would define those details as env vars, eg:

```
  postgres:
    image: postgres:14.0
    environment:
      POSTGRES_USER: postgres
      POSTGRES_PASSWORD: postgres
      POSTGRES_DB: postgres
    ports:
      - 5432:5432
```

I think that going with `postgres` as the defaults is a good idea!